community/graphdb-api/src/main/java/org/neo4j/graphdb/Node.java

```diff
@@ -78,7 +78,9 @@
      * are attached to this node, an empty iterable will be returned.
      *
      * @return all relationships attached to this node
+     * @deprecated this method should return {@link ResourceIterable} instead
      */
+    @Deprecated
```

Comment:
I wonder if there are better ways than marking these methods as deprecated. I'd guess doing this would cause some confusion as to what to use instead.